python/astro_metadata_translator/headers.py

```diff
@@ -354,17 +358,19 @@ def _find_from_resource(
     """
     if package is not None and resource_root is not None:
```

Comment:
It seems strange that these are optional but nothing is done if they're not present.

Reply:
it's because this is what `translator.resource_root` returns if there are no correction resources. Are you saying this should be the responsibility of the caller and this method should raise if it's not given a package and root?